README.md

```diff
@@ -144,21 +144,31 @@ With a block.
 
 ```ruby
 client.query(ids: [42]) do
-  query(:$ids => :'[Int]') do
-    invoices(ids: :$ids) do
+  query(ids: [:int]) do
+    invoices(ids: :ids) do
       id
       fee_in_cents
     end
   end
 end
 ```
+Graphlient supports following Scalar types for parameterized queries by default:
```

Comment:
Another possibility could be to use actual Ruby types, such as `Int` or `String`? Future improvement?